### PR TITLE
Add KING

### PR DESCRIPTION
**Pull request by mtekman**
FOR CONTRIBUTOR:
* [ ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [X] - This PR adds a new tool or tool collection
* [ ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

This is an extension of the initial PR #3106 from @cvravi9

### CODE REVIEW

File: tools/king/king.xml [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+        <param argument="--projection" type="integer" min="0" optional="true" label="Projection N" help="Includes the first N samples of a subset." />
+        <param argument="--unrelated" type="boolean" truevalue="--unrelated" falsevalue="" label="Unrelated Option" help="Extract a list of unrelated individuals." />
+        <param argument="--build" type="boolean" truevalue="--build" falsevalue="" label="Reconstruct Pedigree" help="Reconstructs pedigrees using SNP data" />
+        <param argument="--cluster" type="boolean" truevalue="--cluster" falsevalue="" label="Cluster Parameter" help="Clusters relatives into families by generating an updateid file." />
+        <param argument="--rplot" type="boolean" truevalue="--rplot" falsevalue="" label="R Code and Plots" help="Generates R code first and then calls R program to make plots in a PDF file." />
+    </inputs>
+    <outputs>
+        <data name="kingoutlog" format="txt" label="${tool.name} on ${on_string} : Log" />
+        <collection name="kingoutput" type="list" label="${tool.name} on ${on_string}: Data">
```
[the remark this answers is not in the document]
Comment (bgruening) [reply tools/king/king.xml:64]: But why not seperating the pdf, log ad txt files at least? and maybe make the log files optional?

**Review by bgruening — COMMENTED** [review]
Ready, just my one question! Thanks!

File: tools/king/king.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (mtekman) [reply tools/king/king.xml]: Good point -- I've split the output into three collections

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @mtekman!

**Pull request merged**